Source/missiles.cpp

```diff
@@ -420,7 +420,7 @@ void GetMissileVel(int i, int sx, int sy, int dx, int dy, int v)
 		dxp = (dx + sy - sx - dy) * (1 << 21);
 		dyp = (dy + dx - sx - sy) * (1 << 21);
 		dr = sqrt(dxp * dxp + dyp * dyp);
-		missile[i]._mixvel = (dxp * (v << 16)) / dr;
+		missile[i]._mixvel = (dxp * (v * 65536)) / dr;
 		missile[i]._miyvel = (dyp * (v << 15)) / dr;

```

Comment:
Missed

Reply:
I would consider this as a legit bitshift, no?

Reply:
Reverted this change.